ansible/roles/haproxy/tasks/config.yml

```diff
@@ -125,6 +125,31 @@
   notify:
     - Restart haproxy container
 
+- name: Checking if haproxy-extra certificates dir exists
+  run_once: true
+  stat:
+    path: "{{ haproxy_extra_cert_directory }}"
+  delegate_to: localhost
+  register: haproxy_extra_cert_dir
+  changed_when: false
+
+- name: Copying over haproxy-extra certificates
+  vars:
+    service: "{{ haproxy_services['haproxy'] }}"
+  copy:
+    src: "{{ haproxy_extra_cert_directory }}/"
+    dest: "{{ node_config_directory }}/haproxy/certs/"
+    mode: "0660"
```

Comment:
Why 0660 / 0770?

Reply:
Certs are given u+g rw perms, same as haproxy.pem.
Directories are given u+g rwx, the same perms as `{{ node_config_directory }}/haproxy` and other config dirs.
Arguably these could be tightened but are at worst no less secure than `{{ node_config_directory }}/`.